### PR TITLE
Post-release version bump

### PR DESCRIPTION
**Pull request by jasmezz**
Post-release version bump 2.0.0 -> 2.1.0dev

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/funcscan/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/funcscan _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by maxulysse** [comment]
Not mentionning this PR in the CHANGELOG?

**Review by fellen31 — DISMISSED** [review]

**Comment by fellen31** [comment]
Ah, missed Maximes comment.

**Comment by jasmezz** [comment]
> Not mentionning this PR in the CHANGELOG?

Nope, so far we didn't include version bumps in the changelog. Only real updates like module/tools versions.

**Comment by jasmezz** [comment]
@maxulysse Opinion or approval please? :innocent:

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,16 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v2.1.0 - [unreleased]
+
+### `Added`
+
+### `Fixed`
```
Comment (maxulysse) [inline CHANGELOG.md:10]:
```suggestion
```suggestion
- [#416](https://github.com/nf-core/funcscan/pull/416/) - Post-release version bump

```### `Fixed`
```

Comment (maxulysse) [reply CHANGELOG.md:10]: but if you don't include this usually it's a :+1: from me then

**Review by maxulysse — APPROVED** [review]
feel free to add my suggestions or not

**Pull request merged**